internal/v1/handler_test.go

```diff
@@ -651,6 +651,33 @@ func TestGetCloneStatus(t *testing.T) {
 	require.Equal(t, "us-east-2", awsUS.Region)
 }
 
+func TestGetCloneEntryNotFoundResponse(t *testing.T) {
+	id := uuid.New().String()
+	apiSrv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
AFAICT we do not not an apiSrv for this test, it seems to be purely working on the DB from image-builder (in the clones endpoint) so I think the entire block can be removed:
```go
diff --git a/internal/v1/handler_test.go b/internal/v1/handler_test.go
index b22bc2ad..0f007e34 100644
--- a/internal/v1/handler_test.go
+++ b/internal/v1/handler_test.go
@@ -653,19 +653,8 @@ func TestGetCloneStatus(t *testing.T) {
 
 func TestGetCloneEntryNotFoundResponse(t *testing.T) {
        id := uuid.New().String()
-       apiSrv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-               if "Bearer" == r.Header.Get("Authorization") {
-                       w.WriteHeader(http.StatusUnauthorized)
-                       return
-               }
-               require.Equal(t, "Bearer accesstoken", r.Header.Get("Authorization"))
-               w.Header().Set("Content-Type", "application/json")
-               w.WriteHeader(http.StatusNotFound)
-               fmt.Fprint(w, "404 during tests")
-       }))
-       defer apiSrv.Close()
 
-       srv, tokenSrv := startServer(t, &testServerClientsConf{ComposerURL: apiSrv.URL}, nil)
+       srv, tokenSrv := startServer(t, &testServerClientsConf{}, nil)
        defer func() {
                err := srv.Shutdown(context.Background())
                require.NoError(t, err)
```

Reply:
thank you :)